086-archive-canary.md

```diff
@@ -0,0 +1,92 @@
+# Archive Canary project written in Go
```

Comment:
I find this quite confusing that you talk especially here in the title but also in the text about _Archive Canary project written in Go_. That suggests that there is some other Canary that is not going to be archived. But what the proposal really suggests is _Archive the Canary project_, period. This should be clear from the title as well as in the introduction paragraph (line 3).

Reply:
Good point.
I used the `Canary written in Go` often in the past, so that's why I wrote it here as well.
Anyway, I changed it to make it clear. Thanks